src/scripts/Save.ts

```diff
@@ -7,6 +7,13 @@ class Save {
         localStorage.setItem("player", json);
         localStorage.setItem("mine", Mine.serialize());
         localStorage.setItem("settings", Settings.save());
+
+        let saveObject = {};
+
+        saveObject[Underground.saveKey] = Underground.save();
+
+        localStorage.setItem("save", JSON.stringify(saveObject));
```

Comment:
This is how I want to do all saving in the future.

Reply:
`localStorage` is itself an Object. Is it worth creating an extra layer called `save`, when you eventually plan to move everything here? 

Maybe a simpler structure like `localStorage.setItem(Underground.saveKey, Underground.save());`.